documentation/src/main/resources/docinfo/index-docinfo-footer.html

```diff
@@ -0,0 +1,16 @@
+<!-- Google Analytics -->
+<script type="text/javascript">
+	dataLayer = [{'channel' : '{html.google-tag-manager-channel}', 'additional_tracking_code' : '{html.google-analytics-id}'}];
+	(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+		new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+		j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+	})(window,document,'script','dataLayer','{html.google-tag-manager-id}');
+</script>
+<noscript><iframe src="//www.googletagmanager.com/ns.html?id={html.google-tag-manager-id}" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+<!-- /Google Analytics -->
+
+<!-- HibernateDoc.OutdatedContent -->
+<script src="//code.jquery.com/jquery-3.1.0.min.js" integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s=" crossorigin="anonymous"></script>
```

Comment:
That URL looks weird. Should be https://code.jquery.com/jquery-3.1.0.min.js I reckon?